package lightning;

message Ping {
	required int32 pingdata = 1;
}

message Pong {
	required int32 pingdata = 1;
}

message GoodBye {
	required int32 session_id = 1;
}

